common/define-grammar.js

```diff
@@ -262,9 +262,32 @@ module.exports = function defineGrammar(dialect) {
         previous
       ),
 
+      _import_identifier: $ => choice($.identifier, alias('type', $.identifier)),
+
       import_specifier: ($, previous) => seq(
         optional(choice('type', 'typeof')),
-        previous
+        choice(
+        field('name', $._import_identifier),
+        seq(
+          field('name', choice($._module_export_name, alias('type', $.identifier))),
```

Comment:
One more place where we can use $._import_identifier.

Reply:
I think replacing `choice($._module_export_name, alias('type', $.identifier))` with `$._import_identifier` would change the semantics of the grammar because `$._module_export_name` is equivalent to `choice($.string, $.identifier)` not merely `$.identifier`. See https://github.com/tree-sitter/tree-sitter-javascript/pull/234 for more context.

If you'd like, I could factor out `alias('type', $.identifier)` from `choice($.identifier, alias('type', $.identifier))` into its own rule and use it here.

Reply:
You're absolutely right, this is not the same case!